torch/fx/passes/reinplace.py

```diff
@@ -110,7 +111,9 @@ def run_node(self, node: Node):
     def propagate(self, *args):
         self.multi_output_view_nodes = {}
         self.node_counter = -1
-        with FakeTensorMode.push() as mode:
+        curr_mode = _get_torch_dispatch_mode()
```

Comment:
Sorry late to this PR. Is this still used?

Just with current discussions on modes, if it is I wanted to check what issues we're running into/what patterns show up a lot

Reply:
nope thanks - I just forgot to delete it